unit-testing/test/UnitTesting/Assertions.cs

```diff
@@ -0,0 +1,85 @@
+using System.Dynamic;
+
+namespace UnitTesting;
+
+public class Assertions : Spec
+{
+    [Test]
+    public void Objects_are_equal_assertion()
+    {
+        double expected = 1.0;
+
+        double actual = Math.Floor(1.5);
+
+        actual.ShouldBe(expected);
+    }
+
+    [Test]
+    public void Throws_exception_message_assertion()
+    {
+        Action action = () => throw new Exception("test");
+
+        action.ShouldThrow<Exception>().Message.ShouldBe("test");
+    }
+
+    [Test]
+    public void Throws_exception_assertion()
+    {
+        Action action = () => throw new ArgumentNullException();
+
+        action.ShouldThrow<ArgumentNullException>();
+    }
+
+    [Test]
+    public void Null_check_assertion()
+    {
+        string? actual = null;
+
+        actual.ShouldBeNull();
+    }
+
+    [Test]
+    public void Boolean_assertion()
+    {
+        bool actual = true;
+
+        actual.ShouldBeTrue();
+    }
+
+    [Test]
+    public void Collection_assertion()
+    {
+        var figures = GiveMe.Figures();
+
+        figures.Count.ShouldBe(10);
+        figures.ShouldBeUnique();
+        figures.ShouldAllBe(f => f > -1 && f < 10);
+        figures.ShouldContain(0);

```

Comment:
it is not clear why these assertions should pass. give me part should be something like `GiveMe.Figures(min: 0, max: 9)`